File: solutions/ChitiwanCr/01.py
def main(userInput):
    if(userInput>100 or userInput<1):
        print(userInput)
        return
    if(userInput%3 == 0 and userInput%5 == 0):
        print('FizzBuzz')
    elif(userInput%5 == 0):
        print('Buzz')
    elif(userInput%3 == 0):
        print('Fizz')
    else:
        print(userInput)

if __name__ == "__main__":
    userInput = int(input("Number 1-100: "))
    main(userInput)

File: solutions/khunathorn_thongnopphakhun/01.php
<?php
  
$handle = fopen("php://stdin","r");

echo "Input Number 1 - 100: ";
$input = fgets($handle);
$input = trim($input);

if($input >= 1 && $input <=100):
  for($i=$input; $i<=100; $i++):
    echo ($i%15 == 0) ? "FizzBuzz \n" : null;
    echo ($i%3 == 0) ? "Fizz \n" : $input."\n";
    echo ($i%5 == 0) ? "Buzz \n" : $input."\n";
  endfor;
else:
  echo 'input fail';
endif;